common/main/root.xml

```diff
@@ -4248,6 +4248,9 @@ for derived annotations.
 			<compoundUnit type="per">
 				<compoundUnitPattern>{0}/{1}</compoundUnitPattern>
 			</compoundUnit>
+            <compoundUnit type="times">
+                <compoundUnitPattern>{0}⋅{1}</compoundUnitPattern>
+            </compoundUnit>
```

Comment:
Should use tabs here, not spaces, for consistency with this and other CLDR xml files.

Reply:
I just changed them in my source, and committed, then squashed. But that didn't fix the problem. Since we are going to run the formatter over these anyway, can you just let it go in?

Reply:
Sure